src/runtime/types.h

```diff
@@ -1270,6 +1271,8 @@ class BoxedModule : public Box {
 public:
     DEFAULT_CLASS(module_cls);
 
+    std::vector<Box*> constants;
```

Comment:
I think this is worth a comment here -- I wasn't expecting to find that constants are stored twice on the module, once in the `str/int/long/etc_constants` and once in `constants`.  I think we should eventually move to having the constants be stored on the code objects, but for now I think the current way is fine but should have a comment / todo.